submissions/adeyemiJuwon/HTMLforms/account.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Account create</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="search" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="login.html">Log in</a>
+                </li>
+                <li>
+                    <a href="account.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="contact.html">Contact</a>
+                </li>
+                <li>
+                    <a href="#">Cart</a>
+                </li>
+              </ul>
+    </div>
+    <div>
+        <a href="index.html">
+            <img src="img/mlogo.png" alt="">
+        </a>
+    </div>
+    <hr>
+    <nav>
+        <ul>
+            <li><a href="#">Best Sellers</a></li>
+            <li><a href="#">Baby Care &amp; Wear</a></li>
+            <li><a href="#">Women collection</a></li>
+            <li><a href="#">Electronic Accessiories</a></li>
+            <li><a href="#"> Lights &amp; Lighting</a></li>
+            <li><a href="#">Smart Home &amp; Garden</a></li>
+
+        </ul>
+    </nav>
+</header>
+    <hr> 
+<section>
+  <div>
+    
+      <h1>CREATE ACCOUNT</h1>
+      <p>Please fill in this form to create an account.</p>
+      <form>
+      <label for="email"><b>Email</b></label> 
+      <input type="text" placeholder="Enter Email" name="email"><br>
+      <br>
+      <label for="password"><b>Password</b></label>
+      <input type="password" placeholder="Enter Password" name="password"><br>
+      <br>
+      <label for="paswordRepeat"><b>Repeat Password</b></label>
+      <input type="password" placeholder="Repeat Password" name="passwordRepeat" required><br>
+      <br>
+      <label for="phoneNumber"> Phone Number</label>
+      <input type="number" placeholder="234807686547" name="phoneNubmer"> <br>
+      <br>
+      <label for="adress">Home Adress</label><br> 
```

Comment:
There is a typo while writing `address`.

Reply:
Okay
Thanks